crates/allocator/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/paritytech/ink"
-documentation = "https://substrate.dev/substrate-contracts-workshop/#/"
+documentation = "https://docs.rs/ink_allocator/"
```

Comment:
Just noted that https://docs.rs/ink_allocator/ is completely empty. Maybe at least a module documentation comment would be good?